client/my-sites/site-settings/media-settings.jsx

```diff
@@ -29,10 +31,16 @@ import {
 	isJetpackModuleUnavailableInDevelopmentMode,
 	isJetpackSiteInDevelopmentMode
 } from 'state/selectors';
+import { getMediaStorage } from 'state/sites/media-storage/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { getSitePlanSlug } from 'state/sites/selectors';
+import {
+	getSitePlanSlug,
+	getSiteSlug,
+} from 'state/sites/selectors';
 import { updateSettings } from 'state/jetpack/settings/actions';
+import QueryMediaStorage from 'components/data/query-media-storage';
 import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import PlanStorageBar from 'blocks/plan-storage/bar';
```

Comment:
I would recommend making this it's own block eg something like `<VideoStorage siteId={ siteId } />`